internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-basic-manifest.yml

```diff
@@ -993,6 +997,42 @@ Resources:
       DomainName: !Ref AppDNSName
       PublicAccessDNS: !GetAtt PublicLoadBalancer.DNSName
       PublicAccessHostedZone: !GetAtt PublicLoadBalancer.CanonicalHostedZoneID
+  AppRunnerVpcEndpointSecurityGroup:
```

Comment:
I wonder if we could/should make this and the other VPC endpoint resources more generic so they could potentially be used for other use cases requiring VPC endpoints...?

Reply:
i think we can cross that bridge if/when it comes 😌